docs/cli-arguments.md

```diff
@@ -32,14 +32,14 @@ Usage of ./kube-state-metrics:
   -h, --help                              Print Help text
       --host string                       Host to expose metrics on. (default "0.0.0.0")
       --kubeconfig string                 Absolute path to the kubeconfig file
-      --labels-metric-allow-list string   Allows to pass a list of additional Kubernetes label keys that will be used in the resource' labels metric. By default the metric contains only name and namespace labels. To include additional labels provide a list of resource names in their plural form and Kubernetes label keys you would like to allow for them (Example: '=namespaces=[k8s-label-1,k8s-label-n,...],pods=[app],...)'
       --log_backtrace_at traceLocation    when logging hits line file:N, emit a stack trace (default :0)
       --log_dir string                    If non-empty, write log files in this directory
       --log_file string                   If non-empty, use this log file
       --log_file_max_size uint            Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
       --logtostderr                       log to standard error instead of files (default true)
       --metric-allowlist string           Comma-separated list of metrics to be exposed. This list comprises of exact metric names and/or regex patterns. The allowlist and denylist are mutually exclusive.
       --metric-denylist string            Comma-separated list of metrics not to be enabled. This list comprises of exact metric names and/or regex patterns. The allowlist and denylist are mutually exclusive.
+      --metric-labels-passthrough string  Comma-separated list of additional Kubernetes label keys that will be used in the resource' labels metric. By default the metric contains only name and namespace labels. To include additional labels provide a list of resource names in their plural form and Kubernetes label keys you would like to allow for them (Example: '=namespaces=[k8s-label-1,k8s-label-n,...],pods=[app],...)'
```

Comment:
Either is fine by me, whatever is less confusing for our users. @brancz @tariq1890 thoughts for or against this change? We should get it into rc.1 if we want it.

Reply:
hmm I think I preferred allowlist, but whatever we do, the exact spelling should be consistent with other flags .. I don't feel strongly whether `allow-list` or `allowlist`, but it should be consistent.

Reply:
Sounds good to me as well, @mrueg would you be okay with making the change to the allowlist instead?

Reply:
Thanks for the feedback. I changed it to metric-labels-allowlist and regenerated the docs.